Nosenti/certificate-manager---Innocent
# Task4-KAN-35 Edit Certificate

#### PR description
- This PR is for Editing the information about a certificate

#### Ticket / Jira Link
- [KAN-35](https://task-force-pro-dccs.atlassian.net/jira/software/projects/KAN/boards/1?assignee=712020%3A67b4d316-bd6f-4b16-a625-69afa0078a10&selectedIssue=KAN-35)

#### Checklist:
- [x] I have produced screenshots for the new feature or enhancement

#### How To Test Changes
- Click on Cog Icon in the table and you should see the actions
- Choose edit and you should be redirected to the form with pre-filled data
- Edit any data and save. You should be redirected to the table with data updated

#### Screenshots (Before and After edit):
![image](https://github.com/user-attachments/assets/d52b6b52-552a-451b-966a-b5cf50e9b034)
![image](https://github.com/user-attachments/assets/64e18700-948c-4205-923c-a01f8a078d7f)
